.github/workflows/test.yml

```diff
@@ -73,3 +73,21 @@ jobs:
           run: |
             OSBUILD_TEST_STORE="${{ env.OSBUILD_TEST_STORE }}" \
             tox -e "py36" -- ${{ env.TEST_WORKERS }} test.run.test_assemblers
+
+  # This smoke test runs the unit tests directly on the runner and as a
+  # normal user - it is fast (2min) and should detect obvious issues
+  # (like from pr#1942)
+  unittests_as_user_smoke:
+    name: "Smoke run: unittest as normal user on default runner"
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      # the test_host.py:test_signals_on_separate_fd runs itself but that
+      # run will happen without the tox env so a pip/tox installed pytest
+      # will not be found, install the pytest package as a workaround
+    - run: sudo apt install -y tox python3-pytest
+    - name: "Run as user on default runer"
+      run: |
+        # depsolve tests tend to be slow but are easy to parallize
```

Comment:
I would put this above the `run` like so it's not a comment *in* the commands being run, but that's not really an issue

Reply:
I moved it up and tweaked it a little bit, I hope I did not add more typos though :)